apps/etterlatte-behandling/src/main/kotlin/behandling/migrering/MigreringService.kt

```diff
@@ -18,9 +22,31 @@ class MigreringService(
     private val sakService: SakService,
     private val foerstegangsBehandlingService: FoerstegangsbehandlingService,
     private val behandlingsHendelser: BehandlingHendelserKanal,
-    private val migreringRepository: MigreringRepository
+    private val migreringRepository: MigreringRepository,
+    private val generellBehandlingService: GenerellBehandlingService
 ) {
     fun migrer(request: MigreringRequest) = opprettSakOgBehandling(request)?.let {
+        val pesys = Vedtaksloesning.PESYS.name
+        foerstegangsBehandlingService.lagreKommerBarnetTilgode(
+            it.id,
+            KommerBarnetTilgode(
+                JaNei.JA,
+                "Automatisk importert fra Pesys",
+                Grunnlagsopplysning.Saksbehandler.create(pesys)
```

Comment:
Det er kanskje overkill nå, men hadde det vært naturlig med en egen kildetype for pesys eksplisitt?

Reply:
Høres fornuftig ut syntes jeg 👍  Trodde vi hadde en egen kilde for System også, men den er kanskje fjernet 🤔

Reply:
Vi har eigen type for det som subklasse av `Bruker`, men det er ein annan mekanisme enn denne. Men liker tanken om å innføre pesys eksplisitt også her.